src/recensio/plone/browser/review.py

```diff
@@ -39,7 +39,7 @@ def get_canonical_url(self):
         """XXX.
 
         registry = getUtility(IRegistry)
-        recensio_settings = registry.forInterface(IRecensioSettings)
+        recensio_settings = registry.forInterface(IRecensioSettings, prefix="recensio.plone.settings")
```

Comment:
I think I get you comment about the prefix.
This is not a big deal once you start using plone.api, e.g.:

```
api.portal.get_registry_record("recensio.plone.settings.external_portal_url")
```